### PR TITLE
Hard coded 2 passes in for sample implementation of multiple pass encoding.

### LINKED ISSUE #6: Multi-pass encoding
I took a swing at implementing this myself already, but it was more difficult than i anticipated.

The ability to do multi-pass encoding would help to improve the quality:filesize ratio, though.

### PR DESCRIPTION
**Pull request by ardeaf**
Merge intended to be worked into future settings in pull request #16, simple implementation. Number of passes is hard coded for now at line 84.

This commit closes #6

### CODE REVIEW

File: video_thread.py [hunk 1]
```diff
@@ -79,55 +79,77 @@ def getBackgroundAtIndex(i):
...
+
+    while pass_count < passes:
+        pass_count += 1
+        ffmpegCommand.append('-pass')
+        ffmpegCommand.append(str(pass_count))
+
+        # Reset the progress bar to 0.
+        progressBarValue = 0
+        print("\nStarted encoding pass {} of {}".format(pass_count, passes))
+
```
Comment (ardeaf) [inline video_thread.py]: Everything here at line 96 to line 143 is indented original code, except I updated the string on line 132 & 133

**Comment by IamDH4** [comment]
@ardeaf  The codebase has changed significantly since #16 our current working branch is `feature-newgui` there will most likely be some conflicts with this request. Multipass is a great idea. Are you willing to rebase and port this to the new branch?  Please note however this should be optional and configurable by the user.

**Comment by ardeaf** [comment]
Didn't get a chance to respond, but willing to rebase once I get a better handle on FFMPEG as currently the simple change I did is incompatible with the default parameters we are passing into the ffmpeg command.

**Comment by IamDH4** [comment]
Closing this request for now. Feel free to reopen after a rebase, or submit a new request.

**Pull request closed without merging**